ext/redhat/pe-bolt-server.init

```diff
@@ -37,7 +37,7 @@ fi
 start() {
     echo -n $"Starting PE Bolt Server: "
     mkdir -p $piddir $logdir
-    daemon $daemonopts --user=$owner $exec "${prefix}/config.ru" ${PE_BOLT_SERVER_OPTIONS}
+    daemon $daemonopts --user=$owner $exec -C "${prefix}/puma_config.rb" -d --pidfile $pidfile ${PE_BOLT_SERVER_OPTIONS}
```

Comment:
These changes require https://github.com/puppetlabs/bolt-vanagon/pull/45.

Reply:
Is this just one less tool in the chain? Are there other benefits? Not against it, just curious.

Reply:
Ah, commit on puppetlabs/bolt-vanagon#45 explained. :+1:

Reply:
Using Puma's CLI and a puma config file gives us access to a bunch of options that aren't there using rackup. Puma adds its own layer of config that much overrides rackup. So it's exposing puma config on the CLI, and letting us use its helpers. So daemonization is now `-d --pidfile ...`.